hudi-client/src/main/java/org/apache/hudi/execution/MergeOnReadLazyInsertIterable.java

```diff
@@ -35,8 +36,9 @@
 public class MergeOnReadLazyInsertIterable<T extends HoodieRecordPayload> extends CopyOnWriteLazyInsertIterable<T> {
 
   public MergeOnReadLazyInsertIterable(Iterator<HoodieRecord<T>> sortedRecordItr, HoodieWriteConfig config,
-      String instantTime, HoodieTable<T> hoodieTable, String idPfx) {
-    super(sortedRecordItr, config, instantTime, hoodieTable, idPfx);
+                                       String instantTime, HoodieTable<T> hoodieTable, String idPfx,
```

Comment:
side point: we should fix method arg formatting  consistently between intellij and checkstyle. Keep seeing these sort of whitespace changes in PRs.